### DISCUSSION TITLE
Possibility to create multi window apps?

### DISCUSSION THREAD

**Discussion by arrkiin** [wailsapp/wails#881] [Ideas]
After playing around a bit with wails, I wondered if it is possible to create multi window apps. Or if there are any plans to realized that feature in the future.

**Comment by leaanthony** [comment]
not v2, maybe v3.

**Comment by mondy** [comment]
I would like to know how to achieve multiple windows in the current situation.
Is it possible to call `wails.Run` multiple times in the same program? Or do I have to launch multiple programs that call `wails.Run` only once?

By the way, when I call `window.open` in JavaScript, a new window appears, but I get an `ERR_CONNECTION_REFUSED` error.It appears to be an error because the domain `wails.localhost` does not exist.
How easy is it to get `window.open` to work? If it's easy, can't we use it as a bridge until we get full-fledged multi-window support?
